packages/wrangler/src/secret/index.ts

```diff
@@ -333,7 +349,7 @@ function trimTrailingWhitespace(str: string) {
  * This function can be used to grab the incoming stream of data from, say,
  * piping the output of another process into the wrangler process.
  */
-function readFromStdin(): Promise<string> {
+export function readFromStdin(): Promise<string> {
```

Comment:
same goes with this one 👉  https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/pages/secret/index.ts#L40-L65